tests/roles/run_tests/tasks/upgrade.yml

```diff
@@ -328,7 +328,8 @@
   - name: Upgrade k8s version and boot-image
     include_tasks: "upgrade_k8s.yml"
     loop:
-      - "v1.24.1"
+      - "v1.24.9"
+      - "v1.25.2"
```

Comment:
with this, we are now testing 3 folded upgrades (v1.23>v1.24>v1.25>v1.26) with the same test, maybe we need to think soon about how we would handle v1a5>v1b1 upgrade tests